charts/mlflow/values.yaml

```diff
@@ -1258,6 +1258,17 @@ postgresql:
     existingSecret: ""
   architecture: standalone
   primary:
+    podSecurityContext:
+      enabled: false
+    containerSecurityContext:
+      enabled: false
+      runAsNonRoot: true
+      allowPrivilegeEscalation: false
+      capabilities:
+        drop:
+        - ALL
+      seccompProfile:
+        type: RuntimeDefault
```

Comment:
Yes it does. I added more options in the `containerSecurityContext` but I think it could work just with `.Values.postgresql.primary.containerSecurityContext.enabled` to `false`